README.md

```diff
@@ -230,6 +230,19 @@ def create_menu(title, body, button_text, cancellable):
     # ...
 ```
 
+**Light and good**
+```python
+kargs = {
+    'arg0': 'Hello',
+    'arg1': 'World',
+}
+
+def myfunction(arg0, arg1):
+    print(f'arg0:{arg0} arg1:{arg1}')
+
+myfunction(**kargs)
```

Comment:
It's correct that you can unpack a dict here, but I don't think it's good to use that here. Also no one can say that the keys in the dictionary are named as expected by the function.

Reply:
yes, I agree. I was thinking about a temporal refactoring only.